src/EventListener/Mcw/TinyMce.php

```diff
@@ -63,7 +63,7 @@ public function executeEvent(GetTinyMceStringEvent $event)
         $objTemplate                   = new BackendTemplate('be_' . $file);
         $objTemplate->selector         = 'ctrl_' . $fieldId;
         $objTemplate->type             = $type;
-        $objTemplate->fileBrowserTypes = $fileBrowserTypes;
+        $objTemplate->fileBrowserTypes =  json_encode($fileBrowserTypes);
```

Comment:
This will not work in Contao 4.13. See https://github.com/contao/contao/blob/fb078fd3c1e8b6c81f2743ec01736cf7f70deb55/core-bundle/src/Resources/contao/templates/backend/be_tinyMCE.html5#L44

Reply:
@fritzmg  I understand your concern regarding the behavior in Contao 4.13 and the reference to be_tinyMCE.html5. However, after thorough testing, I can confirm that the fix works as expected.
Below are video recording showing the working behavior in Contao 4.13.

If there's any specific scenario or configuration in Contao 4.13 where this might break, I'd be happy to test further and adjust the implementation if needed!

Looking forward to your thoughts. 😊


https://github.com/user-attachments/assets/7ebc46bd-d740-45e1-ba2c-a1744705e677

Reply:
What is the source code output for this? Wouldn this double json encode the `$fileBrowserTypes`?

Reply:
@fritzmg Could you please review my second commit? I made some changes to fix Contao 4.13 and Contao 5. It works fine; I can upload files and the file picker option is also available.

Reply:
Hm, you are right, the `file_picker_types` setting does not seem to have any discernible effect 🤔 No matter what you put there.

Reply:
@fritzmg i don't get your point but for me contao 4.13 I can see file picker and I uploaded files also see the screenshots below

![image](https://github.com/user-attachments/assets/5e8eb962-0b61-4bbf-ba41-2719a5be0387)
![image](https://github.com/user-attachments/assets/04110a25-dc51-449c-aef6-1fcdc315e87f)
![image](https://github.com/user-attachments/assets/1158322b-5615-4eee-982d-639ac5a37aa0)

Reply:
It turns out that TinyMCE supports both arrays and also space or comma separated strings for that setting - so your current solution works for both Contao 4.13 and 5 👍